### ISSUE TITLE
I cannot enable the "id" on the main view by .gitconfig

### ISSUE THREAD

**Issue by szz** [jonas/tig#766]
I have tried to enable the commit id on the main view by config, but it does not work. 

If I enable id with "o" then "X" keyboard shortcut it works correctly. The id-width property works well from .gitconfig.

I use tig v2.3.0 on MacOS, I have installed it by MacPorts 2.4.2.

**Comment by jonas** [comment]
To enable display of the commit ID in the main view via `~/.tigrc` use:

    set main-view-id = yes

Let me know if that works. Else please provide more info about what config file you are talking about and what setting(s) you have tried.

**Comment by szz** [comment]
Yes, I am able to set id from ~/.tigrc but I would keep my git setting in on place.
I've tried adjust it in the ~/.gitconfig like this:

```
[tig]
    line-graphics = utf-8
    vertical-split = horizontal
    id = yes
    id-width = 10
```

**Comment by jonas** [comment]
This one works:

```
[tig]
        main-view-id = yes
```

**Comment by szz** [comment]
Thank you very much! Works well.

There is a minor misunderstanding because the "id-width" works without full-qualification, other hands I also tried the "main-id" key. So it does not seem too consistent.
